vvm/main.py

```diff
@@ -57,6 +82,9 @@ def compile_source(
     Dict
         Compiler output. The source file name is given as `<stdin>`.
     """
+    if vyper_version is None:
+        vyper_version = detect_vyper_version_from_source(source)
```

Comment:
spooky

Reply:
It's already in the code, why not?

Reply:
we could add a `detect_version` kwarg

Reply:
yea i mean i think it's convenient, but it's a semantics breaking change, downstream users may(?) somehow depend on the existing behavior